tools/blobtoolkit/blobtoolkit.xml

```diff
@@ -165,6 +170,7 @@
                 <option value="create">Create a BlobToolKit dataset</option>
                 <option value="add">Add data to a BlobToolKit dataset</option>
                 <option value="filter">Edit a BlobToolKit dataset</option>
+                <option value="plot">Plot a BlobToolKit dataset</option>
```

Comment:
should the plot not be added __in addition__? I mean create/add/edit a dataset and in addition plot it ...?

Reply:
I think it would be better to keep the plotting option as a seperate functionality

Reply:
And what about both? Adding it as separate functionality AND as option after the other function?

Reply:
Yes I agree! We can ask if the user wants to create/edit/filter **AND** plot the dataset simultaneously. If yes, we can add blobtk to the command and if not then we simply output the dataset. How does this sound?

Reply:
Your call, do you want to add it? The changes look good.

Reply:
I have added the plotting functionality to add/filter options. Now the user has option to plot the blobdataset along with add/filter functionality as well as standalone plot the blobdataset.